### PR TITLE
Show errors

### LINKED ISSUE #7: Show an error if database or application directory does not exist

### PR DESCRIPTION
**Pull request by tetsuo**
Fixes #7

### CODE REVIEW

**Comment by Avaq** [comment]
It looks like this PR includes the commits from #21, let's wait for it to be merged and rebase this.

File: src/cmd.ts [hunk 1]
```diff
@@ -75,19 +77,20 @@ if ((!dir && !argv.version) || argv.help) {
...
                     } else {
-                        console.error('unexpected response: \n' + JSON.stringify(res, null, 2))
+                        console.error('[error] unexpected response: \n' + JSON.stringify(res, null, 2))
                         process.exit(1)
                     }
                 })
-                .catch(e => {
-                    console.error('could not deploy: [' + e.body.error + '] ' + e.message + ' (' + e.body.reason + ')')
-                    throw e
+                .catch(er => {
```
Comment (Avaq) [inline src/cmd.ts]: Remove catch statement and return the Promise

File: src/couchify.ts [hunk 2]
```diff
@@ -84,159 +30,99 @@ const defaultOptions: CouchifyOptions = {
...
-                                })).then(([map, reduce]) => {
-                                    (entry as ViewFunctionResolution).source = {
-                                        map: map && map.slice('module.exports='.length),
-                                        reduce: reduce && reduce.slice('module.exports='.length)
-                                    }
-                                    rewriteResolve(entry)
+                            rewriteRequires(stripJSTags(d.source), name => `./${acc[d.deps[name]]}`)
+                                .then(code => {
+                                    d.source = code
+                                    rewriteResolve(d)
```
Comment (Avaq) [inline src/couchify.ts]: Unwrap from `new Promise`, replace `rewriteResolve` by `return`, remove `catch` statement

**Comment by Avaq** [comment]
Let's squash those commits :fist_right::spiral_notepad::fist_left:

**Comment by tetsuo** [comment]
Closes #14

**Pull request merged**